analyzers/tests/SonarAnalyzer.UnitTest/TestCases/ShiftDynamicNotInteger.CSharp11.cs

```diff
@@ -16,13 +16,8 @@ class ShiftDynamicNotInteger
         public void Test()
         {
             int d = 5;
-            var x = d >>> null; // FN
```

Comment:
Those test cases have been removed since the rule is raising only if the left part of the assignment/expression is a `dynamic` type. Since operators `>>>` and `>>>=` cannot be applied to operands of type `dynamic`, there are no false negatives introduced by this C#11 feature.